community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacadeFactory.java

```diff
@@ -90,6 +90,7 @@
 
         // Kept here to have it not be publicly documented.
         public static final Setting<String> lock_manager = setting( "lock_manager", Settings.STRING, "" );
+        public static final Setting<String> record_format = setting( "record_format", Settings.STRING, "" );
```

Comment:
Doesn't this default mean that you have to explicitly specify "highlimit" to get that format? Or that's just temporary?